test/unit/std/charconv_test.cpp

```diff
@@ -483,3 +504,19 @@ TYPED_TEST(from_char_real_test, non_valid_strings)
         EXPECT_EQ(res.ec, std::errc::invalid_argument);
     }
 }
+
+// =============================================================================
+// std::to_chars for float, double and long double
+// =============================================================================
+
+TYPED_TEST(from_char_real_test, to_chars)
+{
+    TypeParam val{120.3};
+    char buffer[10];
+
+    auto res = std::to_chars(&buffer[0], &buffer[0] + sizeof(buffer), val);
+
+    EXPECT_EQ(res.ptr, &buffer[5]);
+    EXPECT_EQ(res.ec, std::errc{});
+    EXPECT_EQ((std::string_view{buffer, 5}), std::string_view{"120.3"});
```

Comment:
you don't test the case where the buffer is to small?

Reply:
well it is the same function so I thought it is enough to test it once :)